cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/customaction/DefaultCustomActionConfigurer.java

```diff
@@ -42,7 +46,9 @@
   private String description;
   private Map<String, String> properties;
 
-  private DefaultCustomActionConfigurer(CustomAction customAction) {
+  public DefaultCustomActionConfigurer(CustomAction customAction, Id.Namespace deployNamespace, Id.Artifact artifactId,
```

Comment:
oh yes. the constructor can be made private, I thought the comment was about the class. my bad.